sql/records.h

```diff
@@ -71,6 +66,7 @@ struct READ_RECORD
   void    (*unpack)(struct st_sort_addon_field *, uchar *, uchar *);
 
   int read_record() { return read_record_func(this); }
+  uchar *record();
```

Comment:
It should be inline. We don't really want to waste time for function call convention here.